### PR TITLE
Review "About our documentation design" page

### PR DESCRIPTION
**Pull request by jdbaldry**
Closes https://github.com/grafana/writers-toolkit/issues/538.

- Justifies use of first person
- Use correct emphasis for phrases in sentences.

### CODE REVIEW

File: docs/sources/structure/about-documentation-design/index.md [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
   - Grafana
   - documentation
   - page design
+menuTitle: Documentation design
+title: About documentation design
+weight: 100
 ---
 
-# About our documentation design
+# About documentation design
```
Comment (JStickler) [inline docs/sources/structure/about-documentation-design/index.md:16]: Not sure why you removed "our" throughout this topic?  This is not a generic topic, it's specifically about **_Grafana Labs_** documentation design.

Comment (jdbaldry) [reply docs/sources/structure/about-documentation-design/index.md:16]: Good question.  I guess we can revise it to start with "About documentation design at Grafana Labs" or something similar before we and an exception for the use of personal pronouns for the rest of the page and then we would be following the style guidance in https://developers.google.com/style/pronouns#personal-pronouns.

Comment (jdbaldry) [reply docs/sources/structure/about-documentation-design/index.md:16]: I agree, exceptions are very important. I will revise this to start with a clear reference to our organization and then add the exception comment.

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**